packages/cli/src/commands/run.ts

```diff
@@ -126,26 +137,21 @@ export default class RunCommand extends BaseCommand {
   private async runTasks() {
     [this.metaTaskResults, this.metaTaskErrors] = await this.defaultTasks.runTasks();
 
-    if (this.runFlags.task !== undefined) {
-      let pluginTaskResult: TaskResult | undefined;
-      let taskFound: boolean = false;
+    if (this.runFlags.tasks !== undefined) {
+      let { tasksFound, tasksNotFound } = this.pluginTasks.findTasks(this.runFlags.tasks);
```

Comment:
If we change the signature of `findTasks` as per my suggestion below, I think you may need to spread this?

```suggestion
      let { tasksFound, tasksNotFound } = this.pluginTasks.findTasks(...this.runFlags.tasks);
```